libs/domain/picking/offline/services/picking-sync-action-handler.service.ts

```diff
@@ -59,4 +69,35 @@ export class PickingSyncActionHandlerService
       })
     );
   }
+
+  protected handlePush(sync: Sync<PickingSyncAction.Push>): Observable<void> {
+    if (sync.payload.entity !== 'picking-lists') {
+      return throwError(
+        () =>
+          new Error(
+            `PickingSyncActionHandlerService: Unknown ${sync.payload.entity} entity`
+          )
+      );
+    }
+
+    return this.onlineAdapter.get({ ids: sync.payload.ids }).pipe(
+      combineLatestWith(this.indexedDbService.getStore(PickingListEntity)),
+      switchMap(async ([pickingLists, store]) => {
+        // TODO: Remove filtering when BE will be able to retrieve the list of picking lists by provided "ids" list
+        const filteredPickingLists = pickingLists.filter(({ id }) =>
+          sync.payload.ids.includes(id)
+        );
+
+        const addedOrUpdatedKeys = await store.bulkPut(filteredPickingLists, {
+          allKeys: true,
+        });
+
+        if (addedOrUpdatedKeys.length !== filteredPickingLists.length) {
+          throw new Error(
+            `PickingSyncActionHandlerService: Could not add or update all PickingLists after push!`
```

Comment:
@gund Should we schedule another try for the failed ones?

Reply:
No, background sync will automatically do that if the promise is rejected.